docs/source/model_design/general_reference_material/bliss/race_ethnicity/index.rst

```diff
@@ -100,10 +100,137 @@ audience.
 1.1 Research question
 ---------------------
 
-.. todo:: 
+When you’re designing a research question, it’s important to consider both if race, 
+ethnicity, and/or ancestry are relevant, and, if so, what the casual pathways might be. Spending 
+time considering the causal pathways will help you consider what exactly you are 
+investigating and might identify more relevant proxy measures.
+
+To work through this example, let’s consider a starting research question: how will 
+the rates of cardiovascular disease (CVD) change over the next 5 years in the US?
+
+First, let’s consider if including race, ethnicity, and/or ancestry is relevant to 
+the research question. It’s commonly known that CVD rates vary with race, and that 
+race demographics are changing within the US. Understanding how different groups’ 
+CVD rates are changing differently might lead us to make a better forecast for 
+the future. So, it seems relevant to include race here.  
+
+Second, let’s think about if this will lead to changes in policies. Here again, 
+it seems plausible that if we see a sharper increase  in CVD incidence among some 
+racial groups, decision-makers might deploy racially aware intervention strategies. 
+So here, including something about race, ethnicity, and/or ancestry seems relevant. 
+
+Last, let’s think through the possible causal pathways and decide what metric(s) 
+we want to include. In the below diagrams, the solid lines indicate casual pathways, 
+while the dotted lines indicate correlation. 
+
+**Option 1a: Race** 
+
+.. image:: race_a_v1.svg
+
+We stated above that its commonly known that CVD rates vary with race. This is 
+well documented and true. It's easy therefore to think about the casual diagram 
+as a single line between race and disease. But all we know is that there is a 
```

Comment:
```suggestion
as a single arrow from race to disease. But all we know is that there is a 
```
Is this what you mean? Or did you want the arrow to go the other way? Or do you mean a single dotted correlation line?